docs/document/content/quick-start/shardingsphere-scaling-quick-start.cn.md

```diff
@@ -28,11 +28,11 @@ sh %SHARDINGSPHERE_PROXY_HOME%/bin/start.sh
 
 通过相应的 DistSQL 接口管理迁移任务。
 
-详情请参见[使用手册](/cn/user-manual/shardingsphere-scaling/usage/)。
+详情请参见[使用手册] (/cn/user-manual/shardingsphere-scaling/usage/)。
 
 ## 相关文档
 
-- [功能#弹性伸缩](/cn/features/scaling/)：核心概念、使用规范
-- [用户手册#弹性伸缩](/cn/user-manual/shardingsphere-scaling/)：运行部署、使用手册
-- [RAL#弹性伸缩](/cn/user-manual/shardingsphere-proxy/distsql/syntax/ral/#%E5%BC%B9%E6%80%A7%E4%BC%B8%E7%BC%A9)：弹性伸缩的DistSQL
-- [开发者手册#弹性伸缩](/cn/dev-manual/scaling/)：SPI接口及实现类
+- [功能#弹性伸缩] (/cn/features/scaling/)：核心概念、使用规范
+- [用户手册#弹性伸缩] (/cn/user-manual/shardingsphere-scaling/)：运行部署、使用手册
+- [RAL#弹性伸缩] (/cn/user-manual/shardingsphere-proxy/distsql/syntax/ral/ #%E5%BC%B9%E6%80%A7%E4%BC%B8%E7%BC%A9)：弹性伸缩的 DistSQL
```

Comment:
@aishrii Hi, i think this space are unnecessary.

Reply:
@zjcnb, okay, will make necessary changes